DOCS-439: RU review. EN translation. Data storage policies. (#7597)
ClickHouse/ClickHouse: docs/en/operations/table_engines/mergetree.md

        <keep_free_space_bytes>10485760</keep_free_space_bytes>_
    </disk_name_2>
    <disk_name_3>
        <path>/mnt/hdd2/clickhouse</path>
        <keep_free_space_bytes>10485760</keep_free_space_bytes>_
    </disk_name_3>

    ...
</disks>
```

Tags:

- `<disk_name_N>` — Disk name. Names must be different for all disks.
- `path` — path under which a server will store data (`data` and `shadow` folders), should be terminated with '/'.
- `keep_free_space_bytes` — the amount of free disk space to be reserved.

The order of the disk definition is not important.

Storage policies configuration markup:

```xml
<policies>
    <policy_name_1>
        <volumes>
            <volume_name_1>
                <disk>disk_name_from_disks_configuration</disk>
                <max_data_part_size_bytes>1073741824</max_data_part_size_bytes>
            </volume_name_1>
            <volume_name_2>
                <!-- configuration -->
            </volume_name_2>
            <!-- more volumes -->
        </volumes>
        <move_factor>0.2</move_factor>
    </policy_name_1>
    <policy_name_2>
        <!-- configuration -->
    </policy_name_2>

    <!-- more policies -->
</policies>
```

> Тэги:

Reviewer: ```suggestion
Tags:
```

</policies>
```

> In the given example, the `hdd_in_order` policy implements the [round-robin](https://en.wikipedia.org/wiki/Round-robin_scheduling) approach. Since the policy defines only one volume (`single`), the data are stored on all its disks in circular order. Such a policy can be quite useful if there are several similar disks mounted to the system. 

Reviewer: ```suggestion
In given example, the `hdd_in_order` policy implements the [round-robin](https://en.wikipedia.org/wiki/Round-robin_scheduling) approach. Thus this policy defines only one volume (`single`), the data parts are stored on all its disks in circular order. Such policy can be quite useful if there are several similar disks are mounted to the system, but RAID is not configured. Keep in mind that each individual disk drive is not reliable and you might want to compensate it with replication factor of 3 or more.
```

> If there are different disks, the policy `moving_from_ssd_to_hdd` can be used instead. The volume `hot` consists of an SSD disk (`fast_ssd`), and the maximum size of a part that can be stored on this volume is 1GB. All the parts with the size larger than 1GB will be stored directly on the `cold` volume, which contains an HDD disk `disk1`. 

Reviewer: ```suggestion
If there are different kinds of disks available in the system, `moving_from_ssd_to_hdd` policy can be used instead. The volume `hot` consists of an SSD disk (`fast_ssd`), and the maximum size of a part that can be stored on this volume is 1GB. All the parts with the size larger than 1GB will be stored directly on the `cold` volume, which contains an HDD disk `disk1`. 
```
